### PR TITLE
S01 us21 client space

### LINKED ISSUE #23: S01US21_AddClientSpaceController

### LINKED ISSUE #24: S01US21_StyleClientSpace

### PR DESCRIPTION
**Pull request by ghost**
Closes #23 
Closes #24

### CODE REVIEW

**Review by AnthonyBerduck — DISMISSED** [review]
Ok.

**Review by Grefleon — DISMISSED** [review]

**Review by AnthonyBerduck — DISMISSED** [review]
Ok.

File: templates/_layoutDashboard.html.twig [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            <div class="sidebar-brand">
+                <a href="#">Dashboard {{ auth }}</a>
+                <div id="close-sidebar">
+                    <i class="fas fa-times"></i>
+                </div>
+            </div>
+            <div class="sidebar-menu">
+                <ul>
+                    <li class="header-menu">
+                        <span>General</span>
```
Comment (ldupas) [inline templates/_layoutDashboard.html.twig:24]: Général*

**Review by ldupas — CHANGES_REQUESTED** [review]

**Review by OlivierMonsire — APPROVED** [review]

**Review by ldupas — APPROVED** [review]
Ok

**Pull request merged**